src/h.js

```diff
@@ -1,5 +1,10 @@
 var i
 var stack = []
+var context = null
```

Comment:
@acstll This means all users of `h` share the same context. Is that desirable?

Reply:
I think it is, that's the idea. Having access to very common stuff. When you say "users" you mean any module (internal or external) that imports `h`, right?

Reply:
Yes, that's what I meant.